app/browser/api/ledger.js

```diff
@@ -3123,6 +3126,53 @@ const activityRoundTrip = (err, response, body) => {
   updater.checkForUpdate(false, true)
 }
 
+const deleteWallet = (state) => {
+  state = ledgerState.deleteSynopsis(state)
+  state = state.setIn(['settings', settings.PAYMENTS_ENABLED], false)
+
+  client = null
+  synopsis = null
+
+  const fs = require('fs')
+  fs.access(pathName(statePath), fs.constants.F_OK, (err) => {
+    if (err) {
+      return
+    }
+
+    fs.unlink(pathName(statePath), (err) => {
+      if (err) {
+        return console.error('read error: ' + err.toString())
+      }
+    })
+  })
+
+  return state
+}
+
+const clearPaymentHistory = (state) => {
+  state = ledgerState.setInfoProp(state, 'transactions', Immutable.List())
+  state = ledgerState.setInfoProp(state, 'ballots', Immutable.List())
+  state = ledgerState.setInfoProp(state, 'batch', Immutable.List())
+
+  const fs = require('fs')
+  const path = pathName(statePath)
+  try {
+    fs.accessSync(path, fs.constants.W_OK)
+    let data = fs.readFileSync(path)
+    data = JSON.parse(data)
+    if (data) {
```

Comment:
would it be helpful to set `data.memos = []` in here?

Reply:
We would like to keep memos, because we use them for debugging if needed